style/roads.mss

```diff
@@ -1282,6 +1315,27 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
         }
       }
 
+      [construction = 'busway'][zoom >= 15] {
+        line-color: @busway-fill;
```

Comment:
This seems too low contrast vs white. You might need to try:
       `line-color: @minor-construction;`
        `b/line-color: @living-street-fill;`
But there are only 46 contruction=busway segments currently; maybe a dedicated rendering is not needed. https://taginfo.openstreetmap.org/tags/construction=busway

Reply:
> But there are only 46 contruction=busway segments currently; maybe a dedicated rendering is not needed.

That would introduce a weird exception where none is needed.